items.json

```diff
@@ -27227,9 +27227,9 @@
     "name": "Decorated Fullered Sword",
     "culture": "Sturgia",
     "type": "OneHandedWeapon",
-    "price": 7111,
-    "weight": 2.22,
-    "tier": 9.708059,
+    "price": 7588,
+    "weight": 2.7,
+    "tier": 10.0653725,
     "requirement": 0,
     "flags": [

```

Comment:
above tier

Reply:
@namidaka fixed tier in 11.30 push below